go.mod

```
@@ -0,0 +1,12 @@
+module github.com/harness-community/drone-nexus-publish
+
+go 1.12
```

```suggestion
go 1.22.7
```
also, always use Go 1.22.7 as the older versions got some vulns